examples/chip-tool/templates/commands.zapt

```diff
@@ -17,25 +17,24 @@
 {{#chip_client_clusters}}
```

Comment:
As a followup, we should do this for all zcl_clusters, right?

Reply:
Done.